doc/releases/changelog-0.19.1.md

```diff
@@ -0,0 +1,16 @@
+:orphan:
```

Comment:
don't forget to also add this file to `release_notes.md`!

Reply:
Thanks for the catch!